assets/src/data/model/endpoints.js

```diff
@@ -1,32 +1,29 @@
 /**
  * External imports
  */
-import { data, exception as GeneralException } from '@eventespresso/eejs';
-import { isUndefined } from 'lodash';
+import { data } from '@eventespresso/eejs';
 
 /**
- * WP Dependencies
+ * Internal imports
  */
-import { sprintf } from '@eventespresso/i18n';
+import { validateEntityHasKey } from './validators';
```

Comment:
This is another extraction since the logic wrapped by `validateEntityHasKey` was being used in multiple places.  Keeps things DRY.